source/georef/georef.cpp

```diff
@@ -504,9 +504,17 @@ std::vector<nf::Autocomplete<nt::idx_t>::fl_quality> GeoRef::find_ways(const std
         search_str = str;
     }
     if (search_type == 0){
-        to_return = fl_way.find_complete(search_str, nbmax, keep_element, ghostwords);
+        results = fl_way.find_complete(search_str, nbmax, keep_element, ghostwords);
     }else{
-        to_return = fl_way.find_partial_with_pattern(search_str, word_weight, nbmax, keep_element, ghostwords);
+        results = fl_way.find_partial_with_pattern(search_str, word_weight, nbmax, keep_element, ghostwords);
+    }
+
+    // filter with visible parameter
+    for(auto& item: results){
+       Way* way = this->ways[item.idx];
+       if (way->visible == true) {
+            to_return.push_back(item);
+       }
     }
```

Comment:
You have to exclude while adding the address in autocomplete
see: https://github.com/CanalTP/navitia/blob/bace3353ee253b67cb88e6dc38af5b88dc3c971f/source/georef/georef.cpp#L404

Reply:
yes, True. @kinnou02 explained me. 
I transfer the logic. 
It doesn't impact the rest of the PR

Reply:
this is now 

https://github.com/CanalTP/navitia/pull/2592/files#diff-992196e00edd443955062e4bad390541R420